tests/runtest.cmd

```diff
@@ -118,12 +119,6 @@ goto :eof
 set BUILDERRORLEVEL=%ERRORLEVEL%
 goto :eof
```

Comment:
We call to runtests, which if it returns as failing will print a message about where to look for results.  I think removing the dead code is the right option.